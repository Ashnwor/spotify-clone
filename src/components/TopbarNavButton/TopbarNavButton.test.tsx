import React from 'react';
import { shallow } from 'enzyme';
import TopbarNavBtn from './TopbarNavButton';

it('runs correctly for left', () => {
  const wrapper = shallow(<TopbarNavBtn direction="left" />);
  expect(wrapper).toMatchSnapshot();
});

it('runs correctly for right', () => {
  const wrapper = shallow(<TopbarNavBtn direction="right" />);
  expect(wrapper).toMatchSnapshot();
});
