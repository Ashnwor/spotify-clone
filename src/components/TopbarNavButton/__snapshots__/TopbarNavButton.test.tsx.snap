// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly for left 1`] = `
<div
  className="nav-back"
>
  <SvgBack
    className="nav-icon"
  />
</div>
`;

exports[`runs correctly for right 1`] = `
<div
  className="nav-forward"
>
  <SvgForward
    className="nav-icon"
  />
</div>
`;
